src/app-layout/visual-refresh-toolbar/toolbar/trigger-button/styles.scss

```diff
@@ -61,12 +53,12 @@
   justify-content: center;
   align-items: center;
   clip-path: path(
-    'M34.2193 10.1845C33.3961 10.579 32.4739 10.8 31.5 10.8C28.0206 10.8 25.2 7.97939 25.2 4.5C25.2 3.52614 25.421 2.6039 25.8155 1.78066C23.4518 0.639587 20.8006 0 18 0C8.05887 0 0 8.05887 0 18C0 27.9411 8.05887 36 18 36C27.9411 36 36 27.9411 36 18C36 15.1994 35.3604 12.5482 34.2193 10.1845Z'
+    'M32.185 9.34876C31.1913 10.1916 29.905 10.7 28.4999 10.7C25.3519 10.7 22.7999 8.14803 22.7999 5C22.7999 3.68192 23.2473 2.46833 23.9985 1.50281C21.8642 0.537421 19.4948 0 17 0C7.61116 0 0 7.61116 0 17C0 26.3888 7.61116 34 17 34C26.3888 34 34 26.3888 34 17C34 14.2479 33.346 11.6485 32.185 9.34876Z'
   );
 
   @include styles.with-direction('rtl') {
     clip-path: path(
-      'M1.78066 10.1845C2.6039 10.579 3.52615 10.8 4.5 10.8C7.97939 10.8 10.8 7.97939 10.8 4.5C10.8 3.52614 10.579 2.6039 10.1845 1.78066C12.5482 0.639587 15.1994 0 18 0C27.9411 0 36 8.05887 36 18C36 27.9411 27.9411 36 18 36C8.05887 36 0 27.9411 0 18C0 15.1994 0.639587 12.5482 1.78066 10.1845Z'
+      'M1.81502 9.34876C2.80869 10.1916 4.09501 10.7 5.50007 10.7C8.6481 10.7 11.2001 8.14803 11.2001 5C11.2001 3.68192 10.7527 2.46833 10.0015 1.50281C12.1358 0.537421 14.5052 0 17 0C26.3888 0 34 7.61116 34 17C34 26.3888 26.3888 34 17 34C7.61116 34 0 26.3888 0 17C0 14.2479 0.653984 11.6485 1.81502 9.34876Z'
```

Comment:
How is this change related to the rest of the PR?

Reply:
Because the size of the trigger changed. This path is for making sure the red dot badge icon has a gap between itself and the button.

Reply:
Good to know, thanks. For me it sounds something to extract in a separate PR, because it may break screenshot tests (if we would have any for the toolbar mode)

Reply:
These trigger styles are located in the `toolbar/` folder, so I presumed that meant they would only apply to toolbar, which is what we are wanting to change. Everything in this PR would result in a visual change for the toolbar only (on purpose). Why do you say this change be pulled out into a separate PR?

Reply:
Since we moved away from visual contexts, it does not matter much anymore, because it will be whole new code anyway. If you open a new PR with everything and close this one, it is also fine